.circleci/config.yml

```diff
@@ -81,8 +81,8 @@ geth_steps: &geth_steps
           export GOROOT=/usr/local/go
           export GETH_BINARY="$HOME/.py-geth/geth-$GETH_VERSION/bin/geth"
           if [ ! -e "$GETH_BINARY" ]; then
-            curl -O https://storage.googleapis.com/golang/go1.12.1.linux-amd64.tar.gz
-            tar xvf go1.12.1.linux-amd64.tar.gz
+            curl -O https://storage.googleapis.com/golang/go1.9.1.linux-amd64.tar.gz
```

Comment:
Wait... does this PR pass the test suite with go1.12? If it does, we might as well keep it there. If not, 1.9 is good

Reply:
oo good catch!